app/views/admin/talentlms/index.haml

```diff
@@ -6,63 +6,97 @@
 .row
   .col-sm-4
     %h1= content_for :title
-
-.row
-  .col-12.text-right
-    = link_to new_admin_talentlm_path, class: 'btn btn-primary mb-2' do
-      %span.icon-plus
-      Add a Subdomain
-.table-responsive
-  %table.table.table-striped
-    %thead
-      %tr
-        %th Subdomain
-        %th Can Create New Accounts in TalentLMS
-        %th 
-    %tbody
-      - @configs.each do |config|
-        %tr
-          %td
-            = link_to config.unique_name, edit_admin_talentlm_path(config)
-          %td
-            - if config.create_new_accounts 
-              %i.icon-checkmark
-          %td
-            - if config.courses.count == 0 
-              .float-right
-                = link_to admin_talentlm_path(config), method: :delete, data: {confirm: "Are you sure you want to delete the #{config.unique_name} subdomain?"}, class: 'btn btn-sm btn-danger' do
-                  %i.icon-cross
-                  Remove Subdomain
 %br
-%br
-.row
-  .col-12.text-right
-    = link_to new_admin_talentlms_course_path, class: 'btn btn-primary mb-2' do
-      %span.icon-plus
-      Add a Course
-.table-responsive
-  %table.table.table-striped
-    %thead
-      %tr
-        %th Subdomain
-        %th Course Name 
-        %th Course ID
-        %th= "#{length_of_time} to Expiration"
-        %th Default
-        %th
-    %tbody
-      - @configs.each do |config|
-        - config.courses.each do |course|
+.h2 
+  Configuration
+.well
+  = simple_form_for @site_configs, url: update_site_config_admin_talentlms_path(@site_configs), method: :post, remote: true do |f|
+    = f.input :number_lms_courses_required, as: :select_two, collection: { "All": -1, "One of Any": 1 },  input_html: {class: :jRemoteSubmit}, label: 'TalentLMS Courses Completion Requirement'
```

Comment:
Can we move the value for `collection` into the config model?  We should call something like `@site_configs.class.available_completion_counts` (that's probably not quite right).  Having the options in the view makes them harder to enforce and to track down later.

Reply:
Will add this in the next push to the PR

Reply:
This has been included in [this commit](https://github.com/greenriver/hmis-warehouse/pull/4944/commits/8921e9abdb3166418ba55263e027c235c8a0cc52).